Project/Assets/ML-Agents/Examples/Soccer/Scripts/AgentSoccer.cs

```diff
@@ -18,11 +19,27 @@ public enum Team
         Purple = 1
     }
 
+    public enum Position
+    {
+        Striker,
+        Goalie,
+        Generic
```

Comment:
IS this Position used in a scene?

Reply:
It's a dropdown on the soccer agent script

Reply:
But is it selected in one of the scenes? Are we testing that this setting actually learns ?

Reply:
(it is, in the 2v2)